package.json

```diff
@@ -762,15 +762,18 @@
     }
   },
   "scripts": {
-    "vscode:prepublish": "npm run build && npm run markdown",
+    "vscode:prepublish": "npm run build && npm run lint",
```

Comment:
I'd suggest adding `npm run lint` as part of the `build` script. That will make sure we run the linter every time we build the source.
i.e. `"build": "npm run lint && npm run license && tsc --pretty",`

Reply:
Good idea, I've been reorganising a little since too for similar reasons so I'll include that suggestion, cheers.

Reply:
Good thing you suggested that. I needed to add tslint into package.